### ISSUE TITLE
crontab automation does not work on Cheyenne with tcsh

### ISSUE THREAD

**Issue by jwolff-ncar** [ufs-community/regional_workflow#603]
# Description

When I add the following lines to my config.sh
USE_CRON_TO_RELAUNCH="TRUE"
CRON_RELAUNCH_INTVL_MNTS=03

I have an entry in my cron on Cheyenne that looks like:
*/03 * * * * cd /glade/scratch/jwolff/SRW_training/expt_dirs/test_cron && ./launch_FV3LAM_wflow.sh >> ./log.launch_FV3LAM_wflow 2>&1

But when it tries to run I get the error message (in .log.launch_FV3LAM_wflow):
./launch_FV3LAM_wflow.sh: line 113: module: command not found
./launch_FV3LAM_wflow.sh: line 114: module: command not found

It then deletes that line from my cron because it thinks it was successful:
Summary of workflow status:
~~~~~~~~~~~~~~~~~~~~~~~~~~

  0 out of 0 cycles completed.
  Workflow status:  SUCCESS

========================================================================
End of output from script "launch_FV3LAM_wflow.sh".
========================================================================

The end-to-end run of the workflow for the forecast experiment specified 
by expt_name has completed with the following workflow status (wflow_-
status):
  expt_name = "test_cron"
  wflow_status = "SUCCESS"
Thus, there is no need to relaunch the workflow via a cron job.  Remo-
ving from the crontab the line (CRONTAB_LINE) that calls the workflow
launch script for this experiment:
  CRONTAB_LINE = "*/03 * * * * cd /glade/scratch/jwolff/SRW_training/expt_dirs/test_cron && ./launch_FV3LAM_wflow.sh >> ./log.launch_FV3LAM_wflow 2>&1"

## Steps to Reproduce

This may be an environment issue related to using tcsh so these steps would need to be conducted in that environment.

1. Generate an experiment on Cheyenne with the following lines in the config.sh
USE_CRON_TO_RELAUNCH="TRUE"
CRON_RELAUNCH_INTVL_MNTS=03

2. Check the log.launch_FV3LAM_wflow file for errors

**Comment by jwolff-ncar** [comment]
@gsketefian We had some back and forth in Slack (under the SRW App Users' Training workspace) on a couple of options for a possible fix for this. Can we identify how to proceed and get this included?

**Comment by gsketefian** [comment]
From [this exchange](https://ufs-srw-training.slack.com/archives/C02C0GGQYMU/p1632341089120800?thread_ts=1632339234.114600&cid=C02C0GGQYMU) on slack, it looks like the problem is the presence of the `-l` option on the first line of the launch script `launch_FV3LAM_wflow.sh`, i.e.

`
#!/bin/bash -l
`

works, but 

`
#!/bin/bash
`

doesn't because in the latter case, the `module` command is not found.

**Comment by gsketefian** [comment]
This was resolved by PR #675.
